src/devices/Nmea0183/Ais/AisTarget.cs

```diff
@@ -0,0 +1,190 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
```

Comment:
If Ais folder is something you copied from somewhere else please add a comment on top of where was it copied from and give credit to authors (can't remember if this was Ais or something else you mentioned was copied)

Reply:
I did this, but this particular file is entirely from me.